src/test/java/org/folio/rest/impl/TemplateRequestTest.java

```diff
@@ -106,15 +102,14 @@ public static void setUpClass(final TestContext context) throws Exception {
         throw new Exception(message);
     }
 
-    TenantClient tenantClient = new TenantClient(LOCALHOST + ':' + port, TENANT, DUMMY_TOKEN);
+    TenantClient tenantClient = new TenantClient(moduleUrl, TENANT, null);
     DeploymentOptions restVerticleDeploymentOptions = new DeploymentOptions().setConfig(new JsonObject().put("http.port", port));
     vertx.deployVerticle(RestVerticle.class.getName(), restVerticleDeploymentOptions, res -> {
       try {
-        tenantClient.postTenant(null, res2 -> {
-          async.complete();
-        });
+        TenantAttributes t = new TenantAttributes().withModuleTo("mod-template-engine-1.0.0");
+        tenantClient.postTenant(t, res2 -> async.complete());
       } catch (Exception e) {
-        e.printStackTrace();
+        logger.error(e.getMessage());
```

Comment:
This should be replaced by `context.fail(e)`. JUnit will then stop testing and report the message and stacktracae of e.

Reply:
Replaced